overlord/state/change.go

```diff
@@ -423,11 +423,30 @@ func (c *Change) Status() Status {
 	panic(fmt.Sprintf("internal error: cannot process change status: %v", statusStats))
 }
 
+// addNotice triggers a change-update notice.
+func (c *Change) addNotice() error {
+	opts := &AddNoticeOptions{
+		Data: map[string]string{
+			ChangeUpdateNoticeKindDataKey: c.Kind(),
+		},
```

Comment:
I think it makes sense to have it repeat for every occurrence.

On one hand, I think it is a bit spammy but on the other hand it makes sense for listeners that are waiting for a specific change-status like "Done", it would be bad to miss it because it was suppressed through repeat-after.

@pedronis @Meulengracht would love to hear your thoughts.

Reply:
I don't think it's a good idea to have a repeat-after as a general thing for changes. You might miss important change events. Like a change entering error status, done, or waiting.